cmd/service-controller/controller.go

```diff
@@ -905,3 +847,31 @@ func (c *Controller) updateServiceBindings(required types.ServiceInterface, port
 	}
 	return nil
 }
+
+func (c *Controller) getSecret(name string) (*corev1.Secret, error) {
+	return c.vanClient.KubeClient.CoreV1().Secrets(c.vanClient.Namespace).Get(name, metav1.GetOptions{})
+}
+
+func (c *Controller) getConfigMap() (*corev1.ConfigMap, error) {
```

Comment:
Very minor point, really just a personal preference, but getConfigMap() is a little ambiguous. Something like getRouterConfigMap() or getTransportConfigMap() would be more immediately obvious. However it is quite obvious when you look at the definition, so not a real problem.